src/main/scala/Configs.scala

```diff
@@ -38,25 +38,10 @@ class BaseConfig extends Config (
     }
     lazy val globalAddrMap = {
       val memBase = 0x80000000L
-      val memSize = 0x80000000L
-
-      val ioMap = ListBuffer(AddrMapEntry("int", internalIOAddrMap))
-
-      val nMMIOChannels =
-        site(NExtMMIOAXIChannels) +
-        site(NExtMMIOAHBChannels) +
-        site(NExtMMIOTLChannels)
-
-      if (nMMIOChannels > 0) {
-        val extIOBase = 0x60000000L
-        val extIOSize = 0x20000000L
-        ioMap += AddrMapEntry("ext", MemRange(extIOBase, extIOSize, MemAttr(AddrMapProt.RWX)))
-        Dump("IO_BASE", extIOBase)
-        Dump("IO_SIZE", extIOSize)
-      }
-
+      val memSize = 0x10000000L
+      val io = AddrMap((AddrMapEntry("int", internalIOAddrMap) +: site(ExtMMIOPorts).entries):_*)
```

Comment:
You can avoid using the splat by just instantiating the class directly.

    val io = new AddrMap(AddrMapEntry("int", internalIOAddrMap) +: site(ExtMMIOPorts).entries)